utils/pattern/src/frontend/mod.rs

```diff
@@ -214,6 +212,31 @@ where
     }
 }
 
+impl<B, E> Pattern<B, <B::Store as ToOwned>::Owned>
+where
+    B: PatternBackend<StoreUtf8Error = E>,
+    B::Store: ToOwned,
+{
+    /// Creates a pattern from a UTF-8 encoded byte slice.
+    ///
+    /// ✨ *Enabled with the `alloc` Cargo feature.*
```

Comment:
Nit: It does not need the `alloc` feature

Reply:
done